test/jruby/compiler/test_jrubyc.rb

```diff
@@ -15,8 +16,8 @@ def setup; require 'jruby/jrubyc'
 
     @tempfile_stderr = Tempfile.open("test_jrubyc_stderr")
     @old_stderr = $stderr.dup
-    $stderr.reopen @tempfile_stderr
-    $stderr.sync = true
+#    $stderr.reopen @tempfile_stderr
+#    $stderr.sync = true
```

Comment:
Perhaps we can just remove these lines?

Reply:
Actually I suspect these lines were commented out solely for debugging and should be restored. I'll take care of it.